scripts/cnode-helper-scripts/cabal-build-all.sh

```diff
@@ -34,6 +34,14 @@ if [[ "${PWD##*/}" == "cardano-node" ]] || [[ "${PWD##*/}" == "cardano-db-sync"
 	EOF
   chmod 640 cabal.project.local
   cabal install bech32 cardano-addresses-cli  --overwrite-policy=always 2>&1 | tee /tmp/build-b32-caddr.log
+  rm -rf cabal.project.local
+  if [[ "$1" == "-l" ]] ; then
+    cat <<-EOF > cabal.project.local
+		${USE_SYSTEM_LIBSODIUM}
+
+		EOF
+    chmod 640 cabal.project.local
+  fi
 fi
```

Comment:
@rdlrt Is removing the `cabal.project.local` after installing bech32 and cardano-addresses-cli viable? Will `cabal build all` still build the exact tag installed without the entries showing where they were installed from?

Reply:
Yip, absolutely